api/bus.go

```diff
@@ -191,8 +191,10 @@ type UploadParams struct {
 // GougingParams contains the metadata needed by a worker to perform gouging
 // checks.
 type GougingParams struct {
+	ConsensusState     ConsensusState
```

Comment:
we probably need some form of caching on these param types soon, it's not ideal to have every download or upload require all of this information to be fetched from our database and/or other sources